fs/fs_zenfs.cc

```diff
@@ -1120,21 +1119,22 @@ static std::string GetLogFilename(std::string bdev) {
 
   return ss.str();
 }
-#endif
 
 Status NewZenFS(FileSystem** fs, const std::string& bdevname,
                 std::shared_ptr<ZenFSMetrics> metrics) {
   std::shared_ptr<Logger> logger;
   Status s;
 
-#ifndef NDEBUG
   s = Env::Default()->NewLogger(GetLogFilename(bdevname), &logger);
   if (!s.ok()) {
     fprintf(stderr, "ZenFS: Could not create logger");
   } else {
+#ifndef NDEBUG
     logger->SetInfoLogLevel(DEBUG_LEVEL);
-  }
+#else
+    logger->SetInfoLogLevel(INFO_LEVEL);
```

Comment:
I do not want to have a log being recorded in /tmp/ for production builds per default (this is undocumented, and the log messages are essentially debug information) 

Could you make this behavior specific to terarkdb? (make WITH_TERARKDB being set as a prerequisite for setting up the logger and configuring the level)

Reply:
Sounds good to me, we still need to find a way to re-use RocksDB's logger in the future.